src/js/components/dashboard/impacts/DashboardImpactsModal.jsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * DashboardImpactsModal.jsx
+ * Created by Daniel Boos 11/13/19
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import Modal from 'react-aria-modal';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+const propTypes = {
+    data: PropTypes.array,
+    level: '',
+    closeModal: PropTypes.func.isRequired,
+    isOpen: PropTypes.bool
+};
+
+const DashboardImpactsModal = (props) => (
+    <Modal
+        mounted={props.isOpen}
+        onExit={props.closeModal}
+        verticallyCenter
+        underlayClickExits
+        initialFocus="#close-button"
+        titleId="dashboard-page-rule-modal">
+        <div className="usa-da-modal-page">
+            <div id="dashboard-page-rule-modal" className="dashboard-page__modal">
+                <button
+                    id="close-button"
+                    className="close-button"
+                    onClick={props.closeModal}
+                    aria-label="close-modal-button">
+                    <FontAwesomeIcon icon="times" />
+                </button>
+                <h4>Degree of Impact</h4>
```

Comment:
This is more of a design suggestions, but I think it would be helpful to specify whether you're looking at low/medium/high somewhere in the modal.